example/regression-qemu-tfm.job

```diff
@@ -0,0 +1,58 @@
+job_name: 'qemu-tfm-regression'
```

Comment:
@microbuilder, Regarding file naming of the job. How most of existing jobs are named is that project name (the most distinguishing part) is first. Also, most tests are effectively "regression" tests. So, please consider whether word "regression" in the filename adds anything. (It could if e.g. TF-M has, on its side several testsuites, one of which is called "regression" (and other differently)).

So, all in all, I'd suggest to name the job file as `tfm-qemu.job` or `tfm-regression-qemu.job`.

Reply:
Fixed